src/components/Header/SubComponents/CartModal/CartModal.tsx

```diff
@@ -33,6 +39,22 @@ export default function CartModal() {
     handleShowCart()
     router.push('/checkout')
   }
+  const handleAddQty = (cartItem: CartItemType) => () => {
+    handleAddQuantityProduct(cartItem.product)
+  }
+  const handleDeductQty = (cartItem: CartItemType) => () => {
+    handleDeductQuantityProduct(cartItem.product)
+  }
+
+  const handleProductClick = (cartItem: CartItemType) => () => {
+    const product = cartItem.product
+    if (product.category && product.category.__typename !== 'Categories') return
```

Comment:
what is the purpose of this if there is category does this mean when a user click a product it will do nothing?